src/bit-id/bit-id.js

```diff
@@ -131,6 +131,36 @@ export default class BitId {
     throw new InvalidBitId();
   }
 
+  static isValidBitId(id: string) {
```

Comment:
This is not related to this feature (also i don't see you use it). please make a different PR for this.

Reply:
yes ,my mistake , i removed this